Source/WebCore/page/SecurityOrigin.h

```diff
@@ -141,14 +141,17 @@ class SecurityOrigin : public ThreadSafeRefCounted<SecurityOrigin> {
 
     void setStorageBlockingPolicy(StorageBlockingPolicy policy) { m_storageBlockingPolicy = policy; }
 
+    void setShouldBlockDatabaseStorage(bool block) { m_shouldBlockDatabaseStorage = block; }
+    bool shouldBlockDatabaseStorage() { return m_shouldBlockDatabaseStorage; }
```

Comment:
make the method const, as the other ones returning bool, please

Reply:
thanks, fixed